docs/100-upgrade/migration-guides/3.4-3.5.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 3.4 -> 3.5
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.4
+  to 3.5
+---
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/\*@3.5.0"
```

Comment:
**nitpick:** the `\` isn't needed in fact. We've been copy-pasting it for ever without valid reasons I think.

**suggestion:** 
```suggestion
pnpm update "@front-commerce/*@3.5.0"
```

**suggestion(better):** could you look for other occurrences too? I wanted to do it for the release, but it looks like I only did it for the 3.3-3.4 guide

Reply:
Updated on each migration guide